### PR TITLE
Update vpn-conditional-access.md

### PR DESCRIPTION
**Pull request by msarcletti**
Updating the note describing prerequisites for using SSO with information relevant for AAD only joined devices.

### CODE REVIEW

File: windows/security/identity-protection/vpn/vpn-conditional-access.md [hunk 1]
```diff
@@ -77,7 +77,9 @@ Two client-side configuration service providers are leveraged for VPN device com
    - Upon request, forwards the Health Attestation Certificate (received from HAS) and related runtime information to the MDM server for verification
    
 > [!NOTE]
-> Currently, it is required that certificates used for obtaining Kerberos tickets must be issued from an on-premises CA, and that SSO must be enabled in the user’s VPN profile. This will enable the user to access on-premises resources. 
+> Currently, it is required that certificates used for obtaining Kerberos tickets must be issued from an on-premises CA, and that SSO must be enabled in the user’s VPN profile. This will enable the user to access on-premises resources.
+> 
+> In the case of AzureAD-only joined devices (not hybrid joined devices), if the user certificate issued by the on-premises CA has in Subject and SAN (Subject Alternative Name) the user UPN from AzureAD, the VPN profile must be modified to ensure the client does not cache the credentials used for VPN authentication. To do this, after deploying the VPN profile to the client, modify the *Rasphone.pbk* on the client by changing entry **UseRasCredentials** from 1 (default) to 0 (zero).
```
Comment (illfated) [inline windows/security/identity-protection/vpn/vpn-conditional-access.md:82]: The first sentence of the added paragraph appears to have a difficult or unclear sentence structure in the following part:
> if the user certificate issued by the on-premises CA has in Subject and SAN (Subject Alternative Name) the user UPN from AzureAD,

Would it be useful to form a more readable structure by writing, for example, the following?
> if the user certificate issued by the on-premises CA has the user UPN from AzureAD in Subject and SAN (Subject Alternative Name),

Comment (msarcletti) [reply windows/security/identity-protection/vpn/vpn-conditional-access.md]: Thanks for the hint. I've changed the sentence according to your suggestion.

File: windows/security/identity-protection/vpn/vpn-conditional-access.md [hunk 2]
```diff
@@ -77,7 +77,9 @@ Two client-side configuration service providers are leveraged for VPN device com
    - Upon request, forwards the Health Attestation Certificate (received from HAS) and related runtime information to the MDM server for verification
    
 > [!NOTE]
-> Currently, it is required that certificates used for obtaining Kerberos tickets must be issued from an on-premises CA, and that SSO must be enabled in the user’s VPN profile. This will enable the user to access on-premises resources. 
+> Currently, it is required that certificates used for obtaining Kerberos tickets must be issued from an on-premises CA, and that SSO must be enabled in the user’s VPN profile. This will enable the user to access on-premises resources.
+> 
+> In the case of AzureAD-only joined devices (not hybrid joined devices), if the user certificate issued by the on-premises CA has the user UPN from AzureAD in Subject and SAN (Subject Alternative Name) , the VPN profile must be modified to ensure the client does not cache the credentials used for VPN authentication. To do this, after deploying the VPN profile to the client, modify the *Rasphone.pbk* on the client by changing entry **UseRasCredentials** from 1 (default) to 0 (zero).
```
Comment (illfated) [inline windows/security/identity-protection/vpn/vpn-conditional-access.md:82]: Cosmetic/codestyle: comma spacing + adding definite articles (the, that)
```suggestion
> In the case of AzureAD-only joined devices (not hybrid joined devices), if the user certificate issued by the on-premises CA has the user UPN from AzureAD in Subject and SAN (Subject Alternative Name), the VPN profile must be modified to ensure that the client does not cache the credentials used for VPN authentication. To do this, after deploying the VPN profile to the client, modify the *Rasphone.pbk* on the client by changing the entry **UseRasCredentials** from 1 (default) to 0 (zero).
```

**Comment by illfated** [comment]
Just in case my English is not fine-tuned enough to American English writing, I believe we should ask @JohanFreelancer9 to check this content too, to be on the safe side.

**Comment by illfated** [comment]
(also ref. #8612 - similar grammar)

File: windows/security/identity-protection/vpn/vpn-conditional-access.md [hunk 3]
```diff
@@ -77,7 +77,9 @@ Two client-side configuration service providers are leveraged for VPN device com
    - Upon request, forwards the Health Attestation Certificate (received from HAS) and related runtime information to the MDM server for verification
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/vpn/vpn-conditional-access.md:77]:
```suggestion
   - Upon request, forward the Health Attestation Certificate (received from HAS) and related runtime information to the MDM server for verification
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - only @illfated's suggestions and one additional correction to line 72, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Pull request merged**